app/src/main/java/com/conupods/OutdoorMaps/Services/CampusAbstractLocationCreationService.java

```diff
@@ -0,0 +1,66 @@
+package com.conupods.OutdoorMaps.Services;
+
+import com.conupods.OutdoorMaps.Models.Building.AbstractCampusLocation;
+import com.conupods.OutdoorMaps.Models.Building.Building;
+import com.conupods.OutdoorMaps.Models.Building.Campus;
+import com.conupods.OutdoorMaps.Models.Building.Classroom;
+import com.conupods.OutdoorMaps.View.SearchView.AbstractCampusLocationAdapter;
+import com.google.android.gms.maps.model.LatLng;
+
+import java.util.List;
+
+public class CampusAbstractLocationCreationService {
+
+    private List<AbstractCampusLocation> mCampusLocations;
+    private AbstractCampusLocationAdapter mCampusLocationAdapter;
+
+    public CampusAbstractLocationCreationService(List<AbstractCampusLocation> locations, AbstractCampusLocationAdapter campusLocationAdapter) {
+        mCampusLocations = locations;
+        mCampusLocationAdapter = campusLocationAdapter;
+    }
+
+    public AbstractCampusLocationAdapter getCampusLocationAdapter() {
+        return mCampusLocationAdapter;
+    }
+
+    public void prepareCampusLocationsForSearch() {
+
+        //TODO CSV OR JSON PARSER LOGIC TO GO IN HERE INSTEAD OF HARDCODE
+
+        Classroom classroom1 = new Classroom("H_107", new LatLng(73.5790, 45.4973), new Building(null, new LatLng(73.5790, 45.4973), "Hall", new Campus("SGW", new LatLng(73.5790, 45.4973))));
```

Comment:
I thought we were using spaces

Reply:
These are still hardcoded data, I can change them but obviously makes no difference whill connecting everything tommorrow